go/vt/logutil/planetscale_logger.go

```diff
@@ -0,0 +1,47 @@
+package logutil
+
+import (
+	pslog "github.com/planetscale/log"
+	noglog "github.com/slok/noglog"
+
+	"vitess.io/vitess/go/vt/log"
+)
+
+type PSLogger pslog.SugaredLogger
+
+// SetPlanetScaleLogger in-place noglog replacement with PlanetScales' logger.
+func SetPlanetScaleLogger(conf *pslog.Config) (psLogger *pslog.SugaredLogger) {
+	// Use the passed configuration instead of the default configuration
+	if conf != nil {
+		configLogger, _ := conf.Build()
```

Comment:
why are you ignoring the error output? should we not check that before using `configLogger`?